watchtower-plugin/tests/test.py

```diff
@@ -156,3 +156,36 @@ def test_misbehaving_watchtower(node_factory, bitcoind, teosd, directory):
     l1.rpc.pay(l2.rpc.invoice(25000000, "lbl1", "desc1")["bolt11"])
     assert l2.rpc.gettowerinfo(tower_id)["status"] == "misbehaving"
     assert l2.rpc.gettowerinfo(tower_id)["misbehaving_proof"]
+
+
+def test_get_appointment(node_factory, bitcoind, teosd, directory):
+    l1, l2 = node_factory.line_graph(2, opts=[{"allow_broken_log": True}, {"plugin": "watchtower-client"}])
```

Comment:
Is the `allow_broken_log` still needed?

Reply:
It is for `l1` since it does breach a channel.

Notice the format is a list of `dict`, the first one applies to `l1` and the second to `l2`

Reply:
Yeah, IIRC the structure is:

- If there is a single dict, then both nodes share it as conf
- If there is multiple dicts, then each node has its own